### PR TITLE
Adds icon_states to the unused and used Eldritch whetstones

### PR DESCRIPTION
**Pull request by kingofkosmos**
Adds icon_states to the unused (pulsing red glow) and used Eldritch whetstones. Also fixed spans. Thanks to Remie and Cyberboss for the help!
Unused: image
Used: image

:cl:
imageadd: Adds icon_states to the unused and used Eldritch whetstones. Sprites by Fury McFlurry.
/:cl:

### CODE REVIEW

File: code/game/gamemodes/cult/cult_items.dm [hunk 1]
```diff
@@ -161,11 +161,16 @@
...
 	desc = "A block, empowered by dark magic. Sharp weapons will be enhanced when used on the stone."
+	icon_state = "cult_sharpener"
 	used = 0
 	increment = 5
 	max = 40
 	prefix = "darkened"
 
+/obj/item/weapon/sharpener/cult/update_icon()
+	if(used)
+		icon_state = "cult_sharpener_used"
```
Comment (Cyberboss) [inline code/game/gamemodes/cult/cult_items.dm]: icon_state = "cult_sharpener[used ? "_used" : ""]"

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Comment by FuryMcFlurry** [comment]
eldritch stone

Heres the animated not used up whetstone.
Also my Ckey is Fury McFlurry, not FuryMcFlurry

**Comment by kingofkosmos** [comment]
Christ man, first you complain about not getting properly credited in the changelog, only in the PR. And now in the changelog you're still saying it's not properly credited. I dunno what you're achieving with the credits. I mean what are you planning to show these to your future employer or something?

**Comment by FuryMcFlurry** [comment]
Just stating what I want.

**Comment by WJohn** [comment]
Better to have it show whether it's used than not 👍

**Comment by kingofkosmos** [comment]
Checks failed, should I do something about it?

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**